packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -5,10 +5,10 @@
 
 %global gem_name foreman_rh_cloud
 %global plugin_name rh_cloud
-%global foreman_min_version 2.3
+%global foreman_min_version 2.4
```

Comment:
The version in develop has `foreman_min_version 2.5` -- what's correct?

Reply:
This is not what @shimstein said in https://github.com/theforeman/foreman-packaging/pull/6750#discussion_r646116639

I don't mind either way, but can you two please agree ;)

Reply:
I guess it's not the best idea for me to do packaging PR in the evenings :P
updating 👍🏼